command/certificate/install.go

```diff
@@ -172,7 +173,14 @@ func installAction(ctx *cli.Context) error {
 		}
 	}
 
-	ui.Printf("Certificate %s has been installed.", filename)
+	fmt.Printf("Certificate %s has been installed.\n", filename)
```

Comment:
any reason for switching from ui.print here and in the uninstallAction?

Reply:
The actual reason was that `ui.Print` does not exist :)
And I thought, this is the only output of this command, and there's no reason to do it in Stderr (ui always writes there to allow pipes and >)

Reply:
har har. Ok makes sense.